cinema/views.py

```diff
@@ -118,12 +136,17 @@ class OrderPagination(PageNumberPagination):
     max_page_size = 100
 
 
-class OrderViewSet(viewsets.ModelViewSet):
+class OrderViewSet(
+    viewsets.GenericViewSet,
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin
```

Comment:
Mixins are usually placed in the first position in inheritance, follow this approach in the future